python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -159,6 +159,86 @@ def condition():
         ray.get_actor("abc")
 
 
+@pytest.mark.parametrize(
+    "ray_start_cluster_head", [
+        generate_system_config_map(
+            num_heartbeats_timeout=20, ping_gcs_rpc_server_max_retries=60)
+    ],
+    indirect=True)
+def test_create_pg_after_gcs_server_restart(ray_start_cluster_head):
+    cluster = ray_start_cluster_head
+    cluster.add_node(num_cpus=4)
```

Comment:
Can we reduce num_cpus if possible? (Always good to use less processes for testing because VMs only has 2 cpus).